SU2_CFD/src/solvers/CHeatSolver.cpp

```diff
@@ -459,7 +460,7 @@ void CHeatSolver::Viscous_Residual(CGeometry *geometry, CSolver **solver_contain
   su2double laminar_viscosity, Prandtl_Lam, Prandtl_Turb, eddy_viscosity_i, eddy_viscosity_j,
             thermal_diffusivity_i, thermal_diffusivity_j, Temp_i, Temp_j;
 
-  const bool turb = ((config->GetKind_Solver() == MAIN_SOLVER::INC_RANS) || (config->GetKind_Solver() == MAIN_SOLVER::DISC_ADJ_INC_RANS));
```

Comment:
For some reason my IDE decided to change these without me knowing (I must have overlooked it in QC `git diff --staged` ) ... why